website/config/environment.js

```diff
@@ -31,6 +31,9 @@ module.exports = function (environment) {
       siteName: 'Helios Design System',
       title: 'Helios Design System',
       imgSrc: '/assets/logos/share-card.jpg',
+      twitterUsername: '@HashiCorp',
+      // TODO: Add public site URL: https://helios.hashicorp.design
+      url: 'https://hds-website-git-hds-1383-twitter-preview-hashicorp.vercel.app',
```

Comment:
@Dhaulagiri  I added this url temporarily to see if it would work but I'm still not seeing the share image show up. Does the filename of the image (imgSrc: '/assets/logos/share-card.jpg',) get fingerprinted or is there some other reason I'm overlooking?

Reply:
Sorry I overlooked that you could see the fingerprinting in the rendered meta tags so that's not it. Was just trying to think of some possible explanation...